frontend/packages/noobaa-storage-plugin/src/components/bucket-class/modals/_bs-modal.scss

```diff
@@ -0,0 +1,25 @@
+.nb-bc-modal {
+  max-height: 30vh;
+  overflow-y: scroll;
+
+  .modal-body {
+    background-color: #fff;
+  }
+
+  @media screen and (max-width: 426px) {
+    max-height: 73vh;
+  }
+}
+
+.nb-edit-modal__save-btn {
+  margin-right: var(--pf-global--spacer--lg);
+}
+
+.nb-modal {
+  width: 85vw;
+  top: 20vh;
+  @media screen and (max-width: 426px) {
+    top: 3vh;
+    width: unset;
+  }
```

Comment:
Is "modal-content" not working ?

Reply:
Wouldn't `modal-content` alter all the modals?

Reply:
Not asking to override, but use directly that `className`.

Reply:
but when this css gets applied all the modals would get that generic style.